cli/hzn.go

```diff
@@ -758,6 +767,10 @@ Environment Variables:
 		if *allCompBPolFile == "" {
 			allCompBPolFile = allCompDepPolFile
 		}
+
+		if err := version.VerifyExchangeVersion1(exchange.LoadExchangeVersion(*deploycheckOrg, *deploycheckUserPw), false, true); err != nil {
```

Comment:
For `hzn deploycheck` there is a case when deploycheckUserPw is not required when all the inputs are local files. So  please skip the version check if deploycheckUserPw is empty.